package.json

```diff
@@ -145,6 +145,7 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-is": "^17.0.1",
+    "react-router-dom": "^5.2.0",
```

Comment:
Weird to add a package just for tests no? But I agree I don't see how to do different :(

Reply:
i can add a test with Link from @reach/router if necessary also